matrix/lambdas/daemons/v0/driver.py

```diff
@@ -149,3 +165,10 @@ def _add_request_query_to_sqs(self, s3_obj_key: str):
         }
         logger.debug(f"Adding {payload} to sqs {queue_url}")
         self.sqs_handler.add_message_to_queue(queue_url, payload)
+
+    def _fetch_bundle_count_from_analysis_table(self, resolved_bundle_fqids: list):
+        analysis_table_bundle_count_query = analysis_bundle_count_query_template.format(
+            self._format_bundle_fqids(resolved_bundle_fqids))
```

Comment:
FYI I refactored this out of the Driver here: https://github.com/HumanCellAtlas/matrix-service/pull/308/commits/6f103c46a198b9935483bec7fefed7a32af40060